draft-savage-ppm-3phm-mpc.md

```diff
@@ -896,8 +896,8 @@ which is negligible.
 
 The verifiers generate the same point r independently. Each verifier only has
 access to one set of shares from `G(x)` so they each compute a hash of the shares
-they have. They then send that hash to each other, after which they can compute
-the full hash value.
+they have. They then send that hash to each other, after which they can concatenate
+the two hash values and compute commitment hash value.
```

Comment:
Thanks for improving this! The last bit about "compute commitment hash value" is still confusing to me. How about this: "...after which they can concatenate the two hash values and then hash that a final time to generate the challenge point `r`.

Reply:
yea, I stuggled with the end of this too. I used "commitment" since it's used the the pseudo code. The final hash isn't exactly the challenge point, so maybe:

> ... after which they can concatenate the two hash values, hash the concatenation, and proceed with computing the challenge point `r`.

Reply:
Yeah, this is better, but I'm going with:

```suggestion
the two hash values and compute the challenge point.
```